ipykernel/kernelbase.py

```diff
@@ -204,7 +208,7 @@ def _parent_header(self):
         'apply_request',
     ]
     # add deprecated ipyparallel control messages
-    control_msg_types = msg_types + ['clear_request', 'abort_request', 'debug_request']
+    control_msg_types = msg_types + ['clear_request', 'abort_request', 'debug_request', 'usage_request']
```

Comment:
at some point I think this should be a `Enum`, not a request to do that here and now. Maybe just a comment for later.

Reply:
That lambda function is used after by `sum([process_metric_value(process, 'cpu_percent', None) for process in all_processes])` to sum the metric values on all the processes.

As it is written now, it is a sugar and is reused twice. I could rewrite that code block, it is just a matter of code styling I think.

Reply:
I have opened https://github.com/ipython/ipykernel/issues/833 as follow-up